build/commands/scripts/commands.js

```diff
@@ -123,6 +131,10 @@ program
   .option('--bitflyer_staging_client_id <bitflyer_staging_client_id>')
   .option('--bitflyer_staging_client_secret <bitflyer_staging_client_secret>')
   .option('--bitflyer_staging_url <bitflyer_staging_url>')
+  .option('--gemini_wallet_client_id <gemini_wallet_client_id>')
```

Comment:
do any of these actually need to be here? afaik we set this stuff via npm config in CI and all this duplication is a mess as mentioned above. cc @mihaiplesa

Reply:
removed